client/landing/gutenboarding/onboarding-block/question/style.scss

```diff
@@ -69,6 +69,7 @@
 			min-height: 1em;
 			min-width: 60px;
 			transition: color 0.5s ease-in-out 1s, border-color 0.5s ease-in-out 1s;
+			line-height: 1.1; // default is `normal` (20%)
```

Comment:
I'll attempt to address this in another PR that will get aggressive removing a bunch of componentry around here that doesn't make sense any more.